app/src/main/res/values/strings.xml

```diff
@@ -18,17 +18,17 @@
     <string name="error_downloading">Error Downloading %1$s</string>
     <string name="install">Install</string>
     <string name="installing_app">Installing %1$s</string>
-    <string name="installation_aborted">Operation failed because the user aborted the installation.</string>
-    <string name="installation_blocked">Operation failed because the user blocked the installation.</string>
-    <string name="installation_downgrade">Operation failed because the user tried to downgrade the package. Do you have stock YouTube Installed?</string>
-    <string name="installation_conflict">Operation failed because the app conflicts with an already installed app.</string>
-    <string name="installation_failed">Operation failed for an unknown reason, please join our Telegram or Discord for support.</string>
-    <string name="installation_incompatible">Operation failed because the app is incompatible with your device.</string>
-    <string name="installation_invalid">Operation failed because the apk files are invalid.</string>
-    <string name="installation_signature">Operation failed because the user didn\'t disable apk signature verification</string>
-    <string name="installation_miui">An unexpected error occurred while installing Vanced. Have you disabled MIUI Optimisations?</string>
-    <string name="installation_storage">Operation failed due to a storage error.</string>
-    <string name="installation_wait">Please wait until the installation finishes</string>
+    <string name="installation_aborted">Installation failed because user aborted the installation.</string>
+    <string name="installation_blocked">Installation failed because user blocked the installation.</string>
+    <string name="installation_downgrade">Installation failed because user tried to downgrade the package. Uninstall updates from stock YouTube app then try again.</string>
+    <string name="installation_conflict">Installation failed because the app conflicts with an already installed app. Uninstall the current version of Vanced then try again.</string>
```

Comment:
I think "then try again" should be "and try again"

Reply:
Hmm. Both are correct in my opinion.
Let's ask the others.

Reply:
Okay. After some researches, I decided to use , then.
![Screenshot_20200810-000455_Samsung_Internet_Beta](https://user-images.githubusercontent.com/24849056/89737739-acad9880-da9d-11ea-8165-f2510cdfe696.png)